src/lists/_base.scss

```diff
@@ -0,0 +1,9 @@
+/**
+ * @file List
+ * @module List
+ * @overview List and list with action link modules
+ */
+
+.list {
+  padding: ms(1) 0;
```

Comment:
@ckeisato I've been using margin-bottom as a convention for spacing things relative to other modules etc. And padding more for component internal padding etc. The styling for this module will probably change, but it might be a good convention to follow.